src/compiler/rust.rs

```diff
@@ -712,6 +712,9 @@ impl<T> CompilerHasher<T> for RustHasher
                     outputs.insert(dep_info.to_string_lossy().into_owned(), p);
                 }
                 let toolchain_creator = Box::new(RustCompilerPackager { sysroot: sysroot.clone() });
+                let mut arguments = arguments;
+                // Always request color output, the client will strip colors if needed.
+                arguments.push(Argument::WithValue("--color", ArgData::Color("always".into()), ArgDisposition::Separated));
```

Comment:
Would it be simpler to move this into `generate_compile_commands`? I don't remember exactly why I put it here in the first place.